code/modules/mob/living/carbon/xenomorph/castes/lesser_drone.dm

```diff
@@ -118,3 +119,18 @@
 
 /mob/living/carbon/xenomorph/lesser_drone/handle_ghost_message()
 	return
+
+/mob/living/carbon/xenomorph/lesser_drone/handle_screech_act(mob/self, mob/living/carbon/xenomorph/queen/queen)
+	return null
+
+/mob/living/carbon/xenomorph/lesser_drone/handle_queen_screech(mob/living/carbon/xenomorph/queen/queen)
+	to_chat(src, SPAN_DANGER("The mighty roar of the queen makes you tremble and fall over!"))
+	adjust_effect(6, STUN)
+	apply_effect(6, WEAKEN)
+
+/datum/behavior_delegate/lesser_drone_base
+	name = "Base Lesser Drone Behavior Delegate"
```

Comment:
Also not worth having these changes once the hugger changes are resolved. If your fear was lessers face hugging people that gets screeched, it should no longer be a problem. In general I want to promote lessers working in groups, and them being discouraged from hanging around the queen only pushes that goal further away.

Reply:
Health drain when off weeds I will note though I believe does help encourage lessers sticking around other xenos more.

Reply:
My worry here is they would just sit at an egg morpher waiting for screech \ drag people into eggs etc etc

Reply:
If a screech is occuring that close to a Morpher I'm not concerned. Every other xeno ought to be trying to do the same thing in that situation - it's nothing special to lessers.